matlabCode/createBIDS_ieeg_json.m

```diff
@@ -67,82 +83,79 @@
 
-%% General fields, shared with MEG BIDS:
-
-
-
-%%  Required fields:
-
-ieeg_json.EEGChannelCount = ''; % Number of EEG channels included in the recording (e.g. 0) 
+ieeg_json.EEGChannelCount = ''; % Number of scalp EEG channels recorded simultaneously (e.g., 21)
 
-ieeg_json.EOGChannelCount = ''; % Number of EOG channels included in the recording (e.g. 1)
+ieeg_json.EOGChannelCount = ''; % Number of EOG channels
 
-ieeg_json.ECGChannelCount = ''; % Number of ECG channels included in the recording (e.g. 1)
+ieeg_json.ECGChannelCount = ''; % Number of ECG channels
 
-ieeg_json.EMGChannelCount = ''; % Number of EMG channels included in the recording (e.g. 1)
+ieeg_json.EMGChannelCount = ''; % Number of EMG channels
 
-ieeg_json.MiscChannelCount = ''; % Number of miscellaneous channels included in 
-% the recording (e.g. 1)
+ieeg_json.MiscChannelCount = ''; % Number of miscellaneous analog channels for auxiliary  signals
 
-ieeg_json.TriggerChannelCount = ''; % Number of channels for digital (TTL bit level) 
-% triggers (e.g. 0) 
+ieeg_json.TriggerChannelCount = ''; % Number of channels for digital (TTL bit level) triggers. 
 
-ieeg_json.PowerLineFrequency = ''; % Frequency (in Hz) of the power grid where 
-% the iEEG recording was done (i.e. 50 or 60) 
-
-
-
-%% Optional fields:
 ieeg_json.RecordingDuration = ''; % Length of the recording in seconds (e.g. 3600)
 
-ieeg_json.RecordingType = ''; % continuous, epoched 
+ieeg_json.RecordingType = ''; % continuous or epoched 
 
 ieeg_json.EpochLength = ''; % Duration of individual epochs in seconds (e.g. 1).
-% If recording was continuous, set value to Inf.
-
-ieeg_json.DeviceSoftwareVersion = ''; % Manufacturer's designation of the acquisition software.
+% If recording was continuous, leave out the field.
 
 ieeg_json.SubjectArtefactDescription = ''; % Freeform description of the observed 
 % subject artefact and its possible cause (e.g. door open, nurse walked into room at 2 min, 
 % "Vagus Nerve Stimulator", non-removable implant, seizure at 10 min). 
 % If this field is left empty, it will be interpreted as absence of artifacts.
 
+ieeg_json.SoftwareVersions = ''; % Manufacturer's designation of the acquisition software.
+
 
 
 %% Specific iEEG fields:
 
 
+% If mixed types of references, manufacturers or electrodes are used, please
+% specify in the corresponding table in the _electrodes.tsv file
 
 %% Required fields:
-ieeg_json.iEEGSurfChannelCount = ''; % Number of iEEG surface channels included 
-% in the recording (e.g. 120) 
 
-ieeg_json.iEEGDepthChannelCount = ''; % Number of iEEG depth channels included 
-% in the recording (e.g. 8) 
+ieeg_json.iEEGReference = ''; % General description of the reference scheme used and 
+% (when applicable) of location of the reference electrode in the raw recordings 
+% (e.g., "left mastoid�, �bipolar�, �T01� for electrode with name T01, �intracranial electrode
+% on top of a grid, not included with data�, �upside down electrode�).
```

Comment:
add note as in spec: "If different channels have a different reference, this field should have a general description and the channel specific reference should be defined in the _channels.tsv file."